tutorials/audio_tasks/speech_enhancement/Speech_Enhancement_with_NeMo.ipynb

```diff
@@ -0,0 +1,1162 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8a17a741",
+   "metadata": {},
+   "source": [
+    "Now, datasets need to be configured to generate a single-channel input signal and a two-channel output signal.\n",
+    "This can be easily achieved by providing multiple keys for `target_key`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5134ba29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use the same dataset as before\n",
+    "config_dual_output.model.train_ds.manifest_filepath = dataset_manifest['train'].as_posix()\n",
+    "\n",
+    "# The input signal is the same as before\n",
+    "config_dual_output.model.train_ds.input_key = 'noisy_filepath'\n",
+    "\n",
+    "# The target signal is not two channel:\n",
+    "#   the first channel is the clean speech signal\n",
+    "#   the second channel is the noise signal\n",
+    "config_dual_output.model.train_ds.target_key = 'speech_filepath,noise_filepath'\n",
+    "config_dual_output.model.train_ds.min_duration = 0 # load all audio files, without filtering out short files\n",
+    "\n",
+    "# Validation and test datasets are configured in the same way\n",
+    "config_dual_output.model.validation_ds.manifest_filepath = dataset_manifest['test'].as_posix()\n",
+    "config_dual_output.model.validation_ds.input_key = 'noisy_filepath'\n",
+    "config_dual_output.model.validation_ds.target_key = 'speech_filepath,noise_filepath'\n",
+    "\n",
+    "config_dual_output.model.test_ds.manifest_filepath = dataset_manifest['test'].as_posix()\n",
+    "config_dual_output.model.test_ds.input_key = 'noisy_filepath'\n",
+    "config_dual_output.model.test_ds.target_key = 'speech_filepath,noise_filepath'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c544b494",
+   "metadata": {},
+   "source": [
+    "The loss can be easily configured by assigning weights for each output of the model.\n",
+    "For example, equal weight for speech- and noise-related loss can be configured as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "096f8c25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Asign equal weights to speech and noise loss\n",
+    "#   total_loss = 0.5 * speech_loss + 0.5 * noise_loss\n",
+    "config_dual_output.model.loss.weight = [0.5, 0.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b9574a6",
+   "metadata": {},
+   "source": [
+    "Similarly, it is possible to calculate metrics for each output channel separately.\n",
+    "For example, this can be configured as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68b25ade",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup metrics\n",
+    "metrics = {\n",
+    "    # Calculate speech metric using the first channel\n",
+    "    'speech_sisdr': {\n",
```

Comment:
Hmm does it error out if there are more or less channels than number of metrics ?

Reply:
Current design is a bit verbose and repetitive